nowcasting_dataloader/datamodules.py

```diff
@@ -1,11 +1,10 @@
 """Datamodules to use for training"""
 import logging
 import os
-from typing import List, Optional, Tuple, Union
+from typing import Optional
```

Comment:
This change, and the changes below are all just to get the linter to accept the merge with `main`!